BedrockServer.h

```diff
@@ -379,41 +376,22 @@ class BedrockServer : public SQLiteServer {
     atomic<bool> _detach;
 
     // Pointers to the ports on which we accept commands.
-    Port* _controlPort;
-    Port* _commandPort;
+    unique_ptr<Port> _controlPort;
+    unique_ptr<Port> _commandPort;
 
     // The maximum number of conflicts we'll accept before forwarding a command to the sync thread.
     atomic<int> _maxConflictRetries;
 
-    // This is a map of HTTPS requests to the commands that contain them. We use this to quickly look up commands when
-    // their HTTPS requests finish and move them back to the main queue.
-    map<SHTTPSManager::Transaction*, BedrockCommand*> _outstandingHTTPSRequests;
     mutex _httpsCommandMutex;
 
-    // Comparison class to sort command pointers based on their timeout rather than pointer address. This lets us keep
-    // commands ordered such that the first ones to time out are at the front.
-    struct compareCommandByTimeout {
```

Comment:
We only needed this because looking at `_outstandingHTTPSRequests` (which is gone) had to efficiently look through `_outstandingHTTPSCommands` for timed out commands. None of that is required any more, so this isn't either.